pupil_src/shared_modules/calibration_choreography/single_marker_plugin.py

```diff
@@ -56,7 +56,9 @@ class SingleMarkerMode(enum.Enum):
 
     @staticmethod
     def all_modes() -> T.List["SingleMarkerMode"]:
-        return list(SingleMarkerMode)
+        modes = list(SingleMarkerMode)
+        modes = sorted(modes, key=lambda m: m.order)
+        return list(modes)
```

Comment:
So the both suggestions are:

1. Explicit, doing one thing at a time
```suggestion
        modes = list(SingleMarkerMode)
        modes = sorted(modes, key=lambda m: m.order)
        return modes
```

2. one-liner
```suggestion
        return sorted(SingleMarkerMode, key=lambda m: m.order)
```

Reply:
I'd be fine with both.

Reply:
@romanroibu Your call